recipes/libignition-cmake/LICENSE

```diff
@@ -0,0 +1,15 @@
+Software License Agreement (Apache License)
```

Comment:
You can remove this file as conda-build looks for it in the source tarball.

Reply:
Unfortunately, the source tarball for the release we want to use doesn't come with the LICENSE file inside, so that's why I explicitly added LICENSE file here (copying it from a later version).